Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -51,10 +101,54 @@ public static int[] RemoveFirst(int[] array)
         /// <returns>A new array with element removed at a given index. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveAt(int[] array, int index)
         {
-            // ToDo: implement.
+            var newArray = ProcessArray(array, index);
+            return newArray;
+        }
+
+        private static int[] ProcessArray(int[] array, int index)
+        {
+            var isValid = IsValidCase(array, index);
+
+            if (!isValid)
+            {
+                return array;
+            }
+
+            array = Remove(array, index);
             return array;
         }
 
+        private static bool IsValidCase(int[] array, int index)
+        {
+            if (array == null || index < 0 || index == array.Length || array.Length == 0)
```

Comment:
What if index is > array.Length?

Reply:
Then it will caus the wrong output. This one is not catched by tests. 
But if I add it , Codacity is complaining that I have to many or in a line.
Apperentlty 3 is the max.

Reply:
You can format it like this
(a || b || c
      || d || e)